packages/widgets/src/dockpanel.ts

```diff
@@ -1449,6 +1471,11 @@ namespace Private {
     let ar = target.left + target.width - target.x;
     let ab = target.top + target.height - target.y;
 
+    const tabHeight = target.tabBar.node.getBoundingClientRect().height;
```

Comment:
In the `hitTestTabAreas` function above, there is some extra logic to account for border widths when dealing with getBoundingClientRect. Is that needed here? (not sure, just checking)

Reply:
It looks to me like the height (which is the property we are testing) is passing through unmodified:
https://github.com/phosphorjs/phosphor/blob/f16d13f11b433089cccca82edc2e16007be581aa/packages/widgets/src/docklayout.ts#L462-L471

But your point is a good one, @jasongrout, I think we can reuse that data rather than recalculating.

Reply:
Never mind, I misunderstood the return arguments of `hitTestTabAreas`

Reply:
I think the current behavior (without tab area boundaries) is the correct one.